deploy/crds/base/jvmbuildservice.io_systemconfigs.yaml

```diff
@@ -0,0 +1,65 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.6.2
+  creationTimestamp: null
+  name: systemconfigs.jvmbuildservice.io
+spec:
+  group: jvmbuildservice.io
+  names:
+    kind: SystemConfig
+    listKind: SystemConfigList
+    plural: systemconfigs
+    singular: systemconfig
+  scope: Cluster
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: SystemConfig TODO provide godoc description
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            properties:
+              jdk8image:
```

Comment:
It is highly likely we will have more images than these three in future. We should have something like a list of 'tag + image' based structs, they should not be hard coded.

Reply:
I had considered a map a la labels or annotations before "making it simpler"

I'll try that out in the next pass.

Reply:
ok @stuartwdouglas I've updated this to be map based ... ptal